.env.example

```
@@ -35,3 +35,8 @@ RENTAL_EXPIRE_MAX_ITERATIONS=100
 RENTAL_EXPIRE_REQUEST_EXPIRE_MINUTES=15
 
 PUBLIC_ID_SECRET=
+
+FLYWAY_LOCATION=classpath:db/migration/schema,classpath:db/migration/seed
+FLYWAY_ENABLE=true
+FLYWAY_BASELINE_ON_MIGRATE=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FLYWAY_BASELINE_ON_MIGRATE key should go before the FLYWAY_ENABLE key

(UnorderedKey)